include/grpcpp/impl/codegen/server_interface.h

```diff
@@ -20,11 +20,14 @@
 #define GRPCPP_IMPL_CODEGEN_SERVER_INTERFACE_H
 
 #include <grpc/impl/codegen/grpc_types.h>
+//#include <grpcpp/alarm.h>
```

Comment:
Just delete it if you don't want it, rather than commenting.

Reply:
Oops, I was earlier using alarm here to get the tag back on the notification cq, but Alarm wraps the tag with its own tag and so FinalizeResult didn't end up getting called there.